### PR TITLE
[telegram-web-app] popupClosed event button_id parameter

### PR DESCRIPTION
**Pull request by mortifactor**
This PR changes the "popupClosed" event according to official docs (the event has an "button_id" parameter).

- [x ] Provide a URL to documentation or source code which provides context for the suggested changes: [telegram docs link](https://core.telegram.org/bots/webapps#events-available-for-web-apps)

### CODE REVIEW

File: types/telegram-web-app/index.d.ts
Comment (MKRhere) [inline types/telegram-web-app/index.d.ts:141]: According to the docs, this should be `button_id: string | null`, shouldn't it?

> eventHandler receives an object with the single field button_id – the value of the field id of the pressed button. If no buttons were pressed, the field button_id will be null.

**Review by MKRhere — CHANGES_REQUESTED** [review]

File: types/telegram-web-app/index.d.ts
Comment (mortifactor) [reply types/telegram-web-app/index.d.ts:141]: Ah, yes. Sorry, my typescript skills are not perfect 🙄

Comment (MKRhere) [reply types/telegram-web-app/index.d.ts:141]: Please do go ahead and fix it so we can get it merged :)

**Review by KnorpelSenf — APPROVED** [review]
Thank you!

**Review by MKRhere — APPROVED** [review]
LGTM!

**Comment by KnorpelSenf** [comment]
@mortifactor consider adding a test case for your changes. If I'm not mistaken, this will allow us to get these changes published without waiting for a maintainer from DefinitelyTyped to approve them.

**Review by KnorpelSenf — APPROVED** [review]
Great work, thanks!

**Comment by KnorpelSenf** [comment]
Ready to merge

**Pull request merged**
